code/modules/trade/datums/trade_stations_presets/3-rare/cybernetics.dm

```diff
@@ -39,7 +39,12 @@
 			/obj/item/organ/external/robotic/blackshield/l_leg,
 			/obj/item/organ/external/robotic/blackshield/r_leg
 		),
-	)
+		"Hardware Imprinters" = list(
```

Comment:
this file isnt used
Put them in the junkrat or Moe_adv I guess

Reply:
I will leave it up to Wilson since I didn't know he he was reworking stations and he probably knows way better where to shove these.